### DISCUSSION TITLE
What platforms are supported? Can I use Go + GDscript?

### DISCUSSION THREAD

**Discussion by hopesofbuzzy** [quaadgras/graphics.gd#12] [Q&A] [answered]
Can I use both GO and GDscript in one project? Or maybe only use some GO library. How many platforms does it support?

**Comment by Splizard** [comment, marked as answer]
> Can I use both GO and GDscript in one project?

Yes, you can register nodes in Go, and their exported methods/functions using `gd` types will be available to call from GDScript.

> Or maybe only use some GO library

This is also possible but currently requires reasonable knowledge about both Go and Godot, this is still a space to explore and should be easier once there are more examples.

> How many platforms does it support?

I can confirm that the project works on Linux amd64, it should also work on Windows and Mac amd64, I suspect it should work on Mac arm64 as well. Android/MetaQuest is not currently working but I have this on the roadmap.

32-bit systems are not supported and there is currently no support for proprietary consoles. Web export is also tricky to support and not a priority at the moment.
